src/common/Select.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+* @name Select
+* @summary Returns a label and a select
+* @returns Returns a label and a select
+*/
+const Select = props => (
+  <div className='activityCategory'>
+    { /* eslint-disable */ }
```

Comment:
Why is `eslint` disabled?

Reply:
This is cause I was creating reusable drop down component and injecting this component to the form tag later on by passing the data i wanted to be shown on the drop-down ... eslint gave me the Form field must have an associeted control which means it required the label to be in a form tag... but i was reusing this and calling the dropdown from another component hence the reason for disabling it

Reply:
Makes sense, thanks for clarifying